contrib/asan/build.sh

```diff
@@ -34,21 +34,25 @@ JULIA_HOME="$HERE/../../"
 echo
 echo "Installing toolchain..."
 
+# Enable Julia assertions: FORCE_ASSERTIONS=1
+# Enable LLVM assertions:  LLVM_ASSERTIONS=1
+MAKE_ASSERT_FLAGS="FORCE_ASSERTIONS=1 LLVM_ASSERTIONS=1"
```

Comment:
Maybe move this to `Make.user.asan`? (and `Make.user.tools`?)

Reply:
Done! I've added it to both `Make.user.asan` and `Make.user.tools`.

Should I now remove it from `build.sh`?

Reply:
I've removed it from `build.sh`.